example/example.js

```diff
@@ -17,7 +17,10 @@ function main() {
       .join(" > ");
   };
 
-  const picker = new ElementPicker();
+  const picker = new ElementPicker({
+    background: "rgba(153, 235, 255, 0.5)",
+    borderColor: "yellow"
+  });
```

Comment:
I think this needs to be nested under a `style` attribute:

```suggestion
  const picker = new ElementPicker({
    style: {
      background: "rgba(153, 235, 255, 0.5)",
      borderColor: "yellow"
    }
  });
```